From d589ed7b9aa91de94eb558ae83ccebd59b881d61 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Tue, 10 Mar 2020 21:06:29 +0800
Subject: [PATCH] arm64: dts: ls1046a: fix ippdexpcr offset not correct

The wrong offset of ippdexpcr in dtsi causes RCPM driver did not
program ippdexpcr properly, which lead to LPM20 cannot exit by wakeup
source.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1046a.dtsi
@@ -809,7 +809,7 @@
 
 		rcpm: rcpm@1ee208c {
 			compatible = "fsl,ls1046a-rcpm", "fsl,qoriq-rcpm-2.1+";
-			reg = <0x0 0x1ee208c 0x0 0x4>;
+			reg = <0x0 0x1ee2140 0x0 0x4>;
 			#fsl,rcpm-wakeup-cells = <1>;
 		};
 
